src/polymer-project.ts

```diff
@@ -356,6 +374,11 @@ class HtmlRejoiner extends Transform {
     pred.hasAttr('src')
   );
 
+  static isExternalStyle = pred.AND(
```

Comment:
Should be used on line 425 but is missing. Will change it to match a `<link>` tag as well for matching the changes below.